rust_src/src/hashtable.rs

```diff
@@ -165,6 +165,8 @@ impl<'a> Iterator for KeyAndValueIter<'a> {
     }
 }
 
+// The references are necessary to satisfy the lifetime requirements.
+// Otherwise the iterator would be have the copy which would be out of scope.
```

Comment:
Grammar here isn't quite obvious enough to me :)

Reply:
Haha. yeah good catch :-)

Reply:
Clearly I was interrupted while typing that.